git/git.go

```diff
@@ -615,6 +615,32 @@ func GitAndRootDirs() (string, string, error) {
 	return absGitDir, absRootDir, nil
 }
 
+func PathFromRoot(pattern string) ([]string, error) {
```

Comment:
Awesome function: this makes the call site much simpler. I have two suggestions:

1. What do you think about changing the name of this function to be `PathsFromRoot` (plural) since it returns multiple paths?
2. What do you think about adding a `PathFromRoot` (singular) that returns `(string, error)` and additionally returns an error if more than one path matched?

In suggestion 2, I think the idea would be to allow `("", nil)` a valid return value and leave that check (`if len(path) == 0`) at the original location in commands/command_lock.go, but move the `if len(paths) > 1` check to this new function.